scribble-lib/scribble/acmart.rkt

```diff
@@ -179,24 +199,120 @@
       (make-element (make-style "ccsdesc" command-props)
                     (decode-string str))))
 
-(define (title #:short [st #f] str)
-  (if st
-      (make-multiarg-element (make-style "StitleShort" multicommand-props)
-                             (list (decode-string st)
-                                   (decode-string str)))
-      (make-element (make-style "title" command-props)
-                    (decode-string str))))
-      
+(define (prefix->string p)
+  (and p (if (string? p) 
+             (datum-intern-literal p)
+             (module-path-prefix->string p))))
+
+(define (gen-tag content)
```

Comment:
This doesn't need to be here, right? Same for `prefix->string` and the `convert-***` functions.